config_hub/finetune/README.md

```diff
@@ -3,66 +3,67 @@
-|                                   |                        |          |        |                |                  |           |         |                  |      |             |                 |                       |
-| stablelm-base-alpha-3b/full.yaml  | stablelm-base-alpha-3b | Alpaca2k | 1      | 512            | 1                | bf16-true | 4xA10G  | 70.13 min        | $5.6 | 21.23 GB    | 1.513           | 4.540                 |
-| stablelm-base-alpha-3b/lora.yaml  | stablelm-base-alpha-3b | Alpaca2k | 4      | 512            | 1                | bf16-true | 1xA10G  | 13.07 min        | $0.4 | 8.58 GB     | 1.361           | 3.900                 |
-| stablelm-base-alpha-3b/lora.yaml  | stablelm-base-alpha-3b | Alpaca2k | 4      | 512            | 1                | bf16-true | 4xA10G  | 13.16 min        | $1.1 | 8.58 GB     | 1.362           | 3.906                 |
-| stablelm-base-alpha-3b/qlora.yaml | stablelm-base-alpha-3b | Alpaca2k | 4      | 512            | 1                | bf16-true | 1xA10G  | 25.86 min        | $0.8 | 5.24 GB     | 1.388           | 4.009                 |
-| stablelm-base-alpha-3b/qlora.yaml | stablelm-base-alpha-3b | Alpaca2k | 4      | 512            | 1                | bf16-true | 4xA10G  | 25.80 min        | $2.1 | 5.24 GB     | 1.391           | 4.020                 |
-|                                   |                        |          |        |                |                  |           |         |                  |      |             |                 |                       |
-| tiny-llama/full.yaml              | tiny-llama             | Alpaca2k | 1      | 512            | 4                | bf16-true | 1xA10G  | 2.58 min         | $0.1 | 14.10 GB    | 1.088           | 2.968                 |
-| tiny-llama/full.yaml              | tiny-llama             | Alpaca2k | 1      | 512            | 4                | bf16-true | 4xA10G  | 2.57 min         | $0.2 | 14.10 GB    | 1.088           | 2.968                 |
-| tiny-llama/lora.yaml              | tiny-llama             | Alpaca2k | 3      | 512            | 8                | bf16-true | 1xA10G  | 8.09 min         | $0.2 | 13.50 GB    | 1.039           | 2.826                 |
-| tiny-llama/qlora.yaml             | tiny-llama             | Alpaca2k | 3      | 512            | 8                | bf16-true | 1xA10G  | 8.70 min         | $0.3 | 16.24 GB    | 1.056           | 2.874                 |
-| tiny-llama/qlora.yaml             | tiny-llama             | Alpaca2k | 3      | 512            | 8                | bf16-true | 4xA10G  | 8.70 min         | $0.7 | 16.24 GB    | 1.056           | 2.874                 |
+| Config                            | Model                  | Epochs | Max seq length | Micro batch size | Machine | Training runtime | Cost | Peak memory | Validation loss | Validation perplexity | Multitask score |
```

Comment:
Maybe it's simpler to just add MMLU in parentheses here instead of mentioning it in the text above. Up to you.

```suggestion
| Config                            | Model                  | Epochs | Max seq length | Micro batch size | Machine | Training runtime | Cost | Peak memory | Validation loss | Validation perplexity | Multitask score (MMLU) |
```

Reply:
Unfortunately this will create too large headers if in the future we need to write "Multitask score (Average of MMLU + X + Y + Z)".

Reply:
That's fair. How about just using MMLU in the header and then in the text saying that this is a Multitask score and hyperlink the paper (https://arxiv.org/abs/2009.03300) for more info?